bindata/network/ovn-kubernetes/004-config.yaml

```diff
@@ -18,7 +18,7 @@ data:
     apiserver="{{.K8S_APISERVER}}"
 
     [logging]
-    logfile="/dev/stdout"
+    logfile="/var/log/ovnkube.log"
```

Comment:
Casey explicitly changed this to stdout so that oc logs could pick it up.

Reply:
That might be the case for logrus, but not for klog which is already in 4.4 and 4.5. I will check during the stand-up if we have any further plans on performing ovnkube related back-ports to 4.3, and then include the "switch to klog" to 4.3 which would fix this everywhere.